docs/API.md

```diff
@@ -761,6 +762,8 @@ __Parameters__
 |`bucketName` | _string_ | Name of the bucket. |
 |`objectName` | _string_ | Name of the object. |
 |`expiry`     | _number_ | Expiry time in seconds. Default value is 7 days. |
+|`respHeaders`  | _object_ | response headers to override (optional) |
+|`requestDate`  | _Date_ | A date object, the url will be issued at. Default value is now. |
```

Comment:
Title above also needs to be modified accordingly, line# 751:

`### presignedGetObject(bucketName, objectName, expiry[, cb])`
=>
`### presignedGetObject(bucketName, objectName[, expiry, respHeaders, requestDate, cb])`

`(optinal)` information should be added to the above `expiry` and `requestDate` descriptions.